app/jsx/components/InfoPagesComp.jsx

```diff
@@ -81,8 +83,12 @@ class InfoPagesComp extends React.Component {
           {this.props.infoResults.slice(3,12).map( (result) =>
             <ResultComp key={result.id} result={result} />) }
           </div>
-          <PaginationComp formName="facetForm" formButton="facet-form-submit" query={this.props.query} count={this.props.info_count} is_info={true} />
+        {(this.props.info_count > 12) ? 
+          [<PaginationComp key="0" formName="facetForm" formButton="facet-form-submit" query={this.props.query} count={this.props.info_count} is_info={true} />,
+          <button key="1" className="c-infopages__toggle" onClick={()=> this.setState({showMore: false})}>Show fewer results</button>]
```

Comment:
Yes, I know I'm duplicating button code twice, but pagination block was forcing itself into a small width needlessly otherwise, possibly because of my introduction of an additional div wrapper in the Pagination Component in order to handle inclusion of the new hidden field.

Reply:
No problem. However, I'm still seeing some problem with the first row having narrower cards than the subsequent rows, for instance if you expand info results here: http://localhost:4001/search?info_start=36&q=submission

Reply:
Yep, this is a CSS issue I've thrown back into Joel's court.